docs/etl/specializations/README.md

```diff
@@ -0,0 +1,31 @@
+# ETL specializations
+
+This area documents special classes that inherit from ETL base classes to 
+solve specific generic tasks.
+
+## Cache Loader
+
+The Cache Loader is provided as `atc.cache.CacheLoader`. The cached loader 
+passes only some rows on to the write operation, and not all. This can give 
+optimizations for cases where the write operation is costly, such as writes 
+to eventhub or cosmos. 
+
+The following diagram illustrates the flow of data 
+through the cached loader.
+![Cache loader diagram](./cached_loader.svg)
```

Comment:
You could give the image a bagground colour, since it is very hard to read the image when the GitHub is in dark-mode.

Reply:
done